projects/dexalot/index.js

```diff
@@ -0,0 +1,42 @@
+const { sumTokens2 } = require('../helper/unwrapLPs')
+const { get } = require("../helper/http");
+
+const dexalotAPI = "https://api.dexalot.com"
+
+function getTVL(env) {
+  return async (api) => {
+    const mainnetRFQs = await get(`${dexalotAPI}/privapi/trading/deployment?contracttype=MainnetRFQ&env=${env}`)
+    const allTokens = await get(`${dexalotAPI}/privapi/trading/tokens`)
+    const tokens = allTokens.filter((t) => !t.isvirtual && t.env === env).map((t) => t.address)
+    return sumTokens2({ api, owner: mainnetRFQs[0].address, tokens })
+  }
+}
+
+async function getDexalotTVL(_) {
+  const lockedTokens = await get(`${dexalotAPI}/api/stats/orderstvl`)
+  const res = {}
+  for (const {coingecko_id, amount} of lockedTokens) {
+    // Ignore counting tokens without coingecko_id
+    if (coingecko_id == null) {
+      continue
+    }
+    res[coingecko_id] = amount
+  }
+  return res
+}
+
+module.exports = {
+  methodology: "Dexalot TVL is comprised of the token balances locked in the MainnetRFQ (swap) contracts and the value locked in open orders on the Dexalot chain.",
+  arbitrum: {
+    tvl: getTVL("production-multi-arb"),
+  },
+  avax: {
+    tvl: getTVL("production-multi-avax"),
+  },
+  base: {
+    tvl: getTVL("production-multi-base"),
```

Comment:
bit confused about this part, arent tokens first bridged to dexalot, then traded there in the orderbooks and then bridged back? 

My understanding was we will make two listings, one for the bridge, where we count the tokens locked in arbitrum, avax & base, and another for the orderbook

Reply:
ah, got it.

For  3, can you maybe return the tokens addresses on each chain (except dexalot) and compute tvl on chain (via sumTokens2 method)?

overlap between 1 & 2 is fine since we we dont show bridge tvl as part of chain tvl. Also, making a listing for the bridge is optional, we can skip it if you dont want it

Reply:
For 3 we currently do the sumTokens2 inside the getTVL function but instead of hardcoding the token addresses + mainnetrfq address we retrieve this via api. I did it this was way so if we add a new swappable token or have to redeploy a fresh copy of our mainetrfq contract we won't need to create a new PR for it.

We would like to make a listing for the bridge too, should I add a new adapter under `dexalot-bridge/index.js`?